cmd/queue/main.go

```diff
@@ -194,7 +194,7 @@ func main() {
 
 	// If this is set, we run as a standalone binary to probe the queue-proxy.
 	if *readinessProbeTimeout >= 0 {
-		os.Exit(standaloneProbeMain(*readinessProbeTimeout))
+		os.Exit(standaloneProbeMain(time.Duration(*readinessProbeTimeout) * time.Second))
```

Comment:
Maybe rename `readinessProbeTimeout` to `readinessProbeTimeoutSeconds` so it is clear what units are used.
Wouldn't it be cleaner and safer to make `readinessProbeTimeout` a `flag.Duration`?

Reply:
It might be k8s limitation — its params are all seconds, or it might be much bigger change to change how we rewrite the probe to exec probe. In either case outside of scope of this PR :)